src/utils/childProcess.js

```diff
@@ -1,18 +1,14 @@
-import { spawn } from 'child_process';
+import { exec } from 'child_process';
 import Promise from 'bluebird';
 
-const SPACE = ' ';
-
 /**
  * Runs continuous shell process
  * @param {String} cmd Shell command
  * @return {Promise<process>}
  */
 export function runProcess(cmd) {
     return new Promise((resolve, reject) => {
-        const commands = cmd.split(SPACE);
-        const [app, ...args] = commands;
-        const childProcess = spawn(app, args);
+        const childProcess = exec(cmd);
```

Comment:
👍 Thats definitely much simpler. I don't remember now why I've chosen `spawn` (probably had some issues with running images in foreground), but it works!

Reply:
Working on below remark about user input sanitisation, I had further look in node.js childProcess API and I think you used `spawn` without shell because it's secured OOTB, no need to sanitise anything.
Now I'm concerned with forcing change to use `exec`.